crypto/evp/e_aes.c

```diff
@@ -176,7 +176,7 @@ static void ctr64_inc(unsigned char *counter)
 # define HWAES_xts_decrypt aes_p8_xts_decrypt
 #endif
 
-#if     !defined(OPENSSL_NO_ASM) &&                     (  \
+#if     defined(OPENSSL_CPUID_OBJ) &&                   (  \
```

Comment:
OPENSSL_CPUID_OBJ seems to come from configdata.pm (for libcrypto and provider.fips).
Does that mean it is always set?

Reply:
Yes, the rationale is the code below uses OPENSSL_ia32cap_P, therefore the CPUID_OBJ need to be
included.  This fixes build targets without assembler support like the ios, or linux-generic32
which has the same issue, unfortuately I always used linux-generic32 together with no-asm so
I did not notice earlier.

Reply:
Master is ok by the looks of it..